Loops/Loops/Exercise5/Program.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+
+namespace Exercise5
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Enter two words: ");
+            string input1 = Console.ReadLine();
+            string input2 = Console.ReadLine();
+            string dots = "";
+            int dotSum = 30 - (input1.Length + input2.Length);
+
+            for (var i = 0; i < dotSum; i++)
+                dots += ".";
```

Comment:
Vai tad nav metodes, kurai pateikt cik reizes izprintē vienu un to pašu char?

Reply:
Es īsti nevaru neko sameklēt :(  charam dokumentācijā pārsvarā pārbaudes metodes